lib/rest.js

```diff
@@ -27,6 +28,9 @@ module.exports = class MicroStrategyRESTAPI extends RestConnection {
     // https://demo.microstrategy.com/MicroStrategyLibrary/api-docs/index.html#/Reports
     this.reports = new ReportsAPI(this);
 
+    // https://demo.microstrategy.com/MicroStrategyLibrary/api-docs/index.html#/Object%20Management
+    this.objectManagement = new ObjectManagementAPI(this);
```

Comment:
I missed the mark here in the user management PR, but I meant to keep alphabetical order here (same as the api-docs page). Can you move objectManagement and userManagement to the right positions? It's minor now but can quickly get messy while we add the other endpoints.